scripts/rqtl2_wrapper.R

```diff
@@ -0,0 +1,447 @@
+# This script file contains  the an implementation of qtl mapping using r-qtl2
+# For r-qtl1 implementation see: Scripts/rqtl_wrapper.R
+# load the library
+
+library(qtl2)
+library(rjson)
+library(stringi)
+library(stringr)
+library(optparse)
+
+
+
+option_list <- list(
+  make_option(c("-c", "--cores"), type="integer", default=1, help="No of cores to use while making
```

Comment:
why do we default to 1? In xapian, we default to `no_of_cores - 1 ` https://github.com/jnduli/genenetwork3/blob/52cd294c2f1d06dddbd6ff613b11f8bc43066038/scripts/index-genenetwork#L417